navigation/dead_reckoning/sam_dead_reckoning/launch/sam_dr_launch.py

```diff
@@ -4,88 +4,71 @@
 from ament_index_python.packages import get_package_share_directory
 from launch_ros.actions import Node
 
+# TODO xml version of launch file
 
 def generate_launch_description():
-    # Example: using a yaml file for parameters
-    # param_config = os.path.join(
-    #     get_package_share_directory("sam_graph_slam_2"),
-    #     "config",
-    #     "pipeline_config.yaml"
-    # )
-
-    # TODO this is not the correct way to set the name
-    # I think we should talk about how launch files should be set up
-    robot_name = "sam0"
-    odom_topic = f"/{robot_name}/dr/odom"
-    gps_odom_topic = f"/{robot_name}/dr/gps_odom"
+    namespace = "sam0"
+    # These topics are specific to sam dr
```

Comment:
Then we should have a dr_msgs package under messages and a Topics.msg message in there with these topics~
The answer to "what is the topic for X" should be "its defined in messages/X_msgs/Topics.msg".